midp/background.js

```diff
@@ -91,6 +91,7 @@ Native["com/nokia/mid/s40/bg/BGUtils.maybeWaitUserInteraction.(Ljava/lang/String
 
   // If the page is visible, just start the FG MIDlet
   if (!document.hidden) {
+    profile === 3 && startTimeline();
```

Comment:
> Actually, I meant a longer timeout (maybe configurable?) to give the BG MIDlet time to settle.

Heh, that makes a lot more sense. :-)

Indeed, a short timeout gives us numbers that are similar to an actual warm startup. With a 10000ms timeout, I get: 14797, 14019, 14364. With a 5000ms timeout, I get: 14246, 13701, 13171. I've set a 5000ms timeout in cf72a04.